cmd/kube-apiserver/app/server.go

```diff
@@ -95,6 +96,10 @@ func Run(s *options.ServerRunOptions) error {
 	if err := s.CloudProvider.DefaultExternalHost(s.GenericServerRunOptions); err != nil {
 		return fmt.Errorf("error setting the external host value: %v", err)
 	}
+	if err := preflight.RunApiserverChecks(s); err != nil {
```

Comment:
If we want this here, then after all options are validated, i.e. around line 120.